envs/airflow-s3-logging/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3'
+
+services:
+  airflow:
+    image: docker-whirl-airflow:py-${PYTHON_VERSION}-local
```

Comment:
You can actually build from compose, eg:
```
        build:
          context: .
          dockerfile: Dockerfile_airflow
          args:
            AIRFLOW_VERSION: master
            AIRFLOW_EXTRAS: async,celery,crypto,jdbc,hdfs,hive,azure,gcp_api,emr,password,postgres,slack,ssh,atlas
```
You can then rebuild using `docker-compose up --build`